contracts/GringottsBankV2.sol

```diff
@@ -163,8 +163,7 @@ contract  GringottsBank is DSAuth, BankSettingIds {
         require(deposits[_depositID].unitInterest == _unitInterest, "Params error: unitInterest");
         require(deposits[_depositID].value == _value, "Params error: amount");
```

Comment:
_depositor, _months, startAt, unitInterest, value

These params can be removed